src/category_theory/sums.lean

```diff
@@ -0,0 +1,171 @@
+/-
+Copyright (c) 2019 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.equivalence
+import category_theory.eq_to_hom
+
+/-#
+Disjoint unions of categories, functors, and natural transformations.
+-/
+
+namespace category_theory
+
+universes v₁ u₁ -- declare the `v`'s first; see `category_theory.category` for an explanation
+
+open sum
+
+section
+variables (C : Type u₁) [𝒞 : category.{v₁+1} C] (D : Type u₁) [𝒟 : category.{v₁+1} D]
+include 𝒞 𝒟
+
+/--
+`sum C D` gives the direct sum of two categories.
+-/
+instance sum : category.{v₁+1} (C ⊕ D) :=
+{ hom     :=
```

Comment:
Why the big space? Maybe that's just how you roll

Reply:
Fixed. I think this was a leftover from when there were proof fields, and I was aligning the `:=`s.